Tests/NIOHTTP2Tests/HTTP2StreamMultiplexerTests.swift

```diff
@@ -1927,4 +1927,51 @@ final class HTTP2StreamMultiplexerTests: XCTestCase {
         XCTAssertNoThrow(try channel.finish(acceptAlreadyClosed: false))
     }
 
+    @available(*, deprecated, message: "Deprecated so deprecated functionality can be tested without warnings")
```

Comment:
Why do we need this to be deprecated? What deprecated function are we using?

Reply:
Not using any deprecated function to my knowledge; I added this since every other test in this file has it and I thought it's used to avoid warnings caused by potential future deprecations. Will remove it